torq.q

```diff
@@ -344,6 +344,12 @@ $[count[req] = count req inter key params;
 	.lg.o[`init;"ignoring partial subset of required process parameters found on the command line - reading from file"];
   ()];		 
 
+// If parentproctype has been supplied then set it
+parentproctype:`$();
```

Comment:
It could be, in which case I would have to change
``` not null first .proc.parentproctype;.proc.reloadparentprocesscode[]```
to
```not null first `$.proc.parentproctype;.proc.reloadparentprocesscode[]```

as applying null on a general empty list gives a generic null

But I think the cast is neater inside the function instead of on assignment

Reply:
Good point. 
Just leave as is.  It's not that important